src/app/components/pages/send-skycoin/send-verify/send-verify.component.ts

```diff
@@ -25,12 +25,15 @@ export class SendVerifyComponent {
     this.snackbar.dismiss();
     this.sendButton.resetState();
     this.sendButton.setLoading();
+    this.backButton.setDisabled();
 
     this.walletService.injectTransaction(this.transaction.inputs, this.transaction.outputs)
       .subscribe(
         () => this.onSuccess(),
         (error) => this.onError(error)
       );
+
+    this.walletService.loadBalances();
```

Comment:
This should not go here, because it updates the balance before the transaction is sent.

Reply:
Fixed.